### DISCUSSION TITLE
[Question]请问一下, 候选面板上的 comment 是由哪里产生的?

### DISCUSSION THREAD

**Discussion by pdog18** [rime/home#1273] [Q&A] [answered]
我需要查看当候选词面板上有  comment  的界面布局情况(例如色彩, UI  尺寸).
但是我不太明白这些  comment  是由哪里控制的, 是词库吗?(我尝试找文档, 但是没有找到)
我看到似乎一些反查, 或者「拆字」的方案会有这个  comment, 但是像我主要使用「袖珍拼音」的几乎从未看到过这个  comment.

现在我暂时使用的是  Lua  脚本产出的  Comment  来查看调整配色方案后的整体效果

**Comment by MokOopsing** [comment, marked as answer]
袖珍拼音 把方案里的spelling_hints设为true后comment就可以出来了

参考 https://github.com/LEOYoon-Tsaw/Rime_collections/blob/master/Rime_description.md#%E4%B8%89translator

**Comment by pdog18** [comment]
不过，其他方案的规则又是咋样的，看来是由 translator 来定义的

**Comment by MokOopsing** [comment]
形码 enable_completion
